docs/UserGuide.md

```diff
@@ -14,13 +25,25 @@ Intern Hub is a desktop app for managing contacts, optimized for use via a Comma
 
 ## Quick start
 
-1. Ensure you have Java `11` or above installed in your Computer.
+1. **Check Java Version & Install Java**
+    - For this application, your system is required to have Java 11 installed
+    - To check if you have Java installed, open your command prompt or terminal and type : `java -version`
+      - If Java is installed, you will see the version number in the output : `openjdk version "11.0.22" 2024-01-16 LTS`
+    - If Java is not installed or your version does not match as the output above :
+      - Visit the Official [Oracle website](https://www.oracle.com/java/technologies/downloads/#java11) to download jdk-11 & follow the download instructions
+      - For **mac** users, download the jdk-11 from [here](https://www.azul.com/downloads/?version=java-11-lts&os=macos&architecture=arm-64-bit&package=jdk-fx#zulu)
 
 2. Download the latest `addressbook.jar` from [here](https://github.com/se-edu/addressbook-level3/releases).
 
-3. Copy the file to the folder you want to use as the _home folder_ for your AddressBook.
+3. Copy the file to the folder you want to use as the _home folder_ for your InternHub.
+    - The home folder refers to the directory or location on your computer where you want to store the files and data related to InternHub
+    - This folder will serve as the central location where InternHub will read and write its files, such as configuration files, databases, or any other files it may use.
+
+4. Once you have set up the _home folder_, right-click on the _home folder_ and click copy to copy the path to this directory.
 
-4. Open a command terminal, `cd` into the folder you put the jar file in, and use the `java -jar addressbook.jar` command to run the application.<br>
+5. Type `cd ` and paste the _home folder_ path, your command should look something like this : `cd /Users/john/home folder`, then hit enter.
+
```

Comment:
This installation guide provides clear and detailed instructions for users to set up and run the InternHub application. The step-by-step approach makes it easy for users to follow along, especially for those who may not be familiar with Java or command-line operations.